testfixtures/logcapture.py

```diff
@@ -3,11 +3,18 @@
 import logging
 import warnings
 from pprint import pformat
-
 from testfixtures.comparison import compare
 from testfixtures.utils import wrap
 
 
+def ensure_assertions_for():
+    """
+    :return: The minimum log level assertions are required for.
+    :rtype: int
+    """
+    return logging.NOTSET
```

Comment:
Let's lose this function and just have the default be static.
To support your monkeypatching plan, I'd suggest a `default_level_for_required_checks` class attribute on `LogCapture` that's set to `logging.NOTSET`.